src/tinykeys.ts

```diff
@@ -146,12 +146,22 @@ export default function keybindings(
 			}
 		})
 
-		clearTimeout(timer)
+		if (timer) {
+			clearTimeout(timer)
+		}
+
 		timer = setTimeout(possibleMatches.clear.bind(possibleMatches), TIMEOUT)
 	}
 
-	target.addEventListener("keydown", onKeyDown as any)
+	target.addEventListener(
+		"keydown",
+		onKeyDown as EventListenerOrEventListenerObject,
```

Comment:
Can you do this without using `as` it’s not actually type safe

Reply:
Fix, it's okay?)